schematic/store/synapse.py

```diff
@@ -50,6 +50,7 @@
 from schematic.exceptions import MissingConfigValueError, AccessCredentialsError
 
 from schematic import CONFIG
+import datetime
```

Comment:
Nit: `datetime` is a python base packge, so I would add it to the top. ([pep8 standards for package imports](https://peps.python.org/pep-0008/#imports))